gnupg/_parsers.py

```diff
@@ -1027,6 +1033,9 @@ def sub(self, args):
         subkey = [args[4], args[11]]
         self.curkey['subkeys'].append(subkey)
 
+    def rvk(self, args):
+        self.curuid == None
```

Comment:
@isislovecruft @meskio Ok, I remember what happened here. In the original issue, I posted the output of what I thought was the key that was causing the problem.

However, the problem (I *think*) was that the key before the one that was erring out had either a) no subkeys and one revocation cert or b) the last key was revoked. This did something like causing the next key to be parsed to attempt to access a `dict` member that was based off a key that didn't exist.

Because this was something I was doing at an old job, I don't have access to the key in question to share the output (and it may have changed since then anyway).